src/quacc/recipes/espresso/phonons.py

```diff
@@ -342,6 +350,19 @@ def grid_phonon_flow(
         This will reduce the amount of data produced by a factor of nblocks.
         If nblocks = 0, each job will contain all the representations for a
         single q-point.
+    run_relax
+        If True, structure relaxation is performed using [quacc.recipes.espresso.core.relax_job][]
+        before running parallelized phonon jobs.
+    copy_files
+        Source directory or directories to copy files from. If a `SourceDirectory` or a
+        list of `SourceDirectory` is provided, this interface will automatically guess
+        which files have to be copied over by looking at the binary and `input_data`.
+        If a dict is provided, the mode is manual, keys are source directories and values
+        are relative path to files or directories to copy. Glob patterns are supported.
+    prev_outdir
+        The output directory of a previous calculation. If provided, Quantum Espresso
+        will directly read the necessary files from this directory, eliminating the need
+        to manually copy files. The directory will be ungzipped if necessary.
```

Comment:
We probably want to think about how to clarify the language here. For instance, if `run_relax == True`, the user-specified `prev_outdir` will do nothing because the `prev_outdir` used in `ph_init_job` is the `pw_job` relaxation job that was run immediately beforehand. Perhaps we can specify in the docstring that this `prev_outdir` keyword argument is specifically used in the call to `ph_init_job` in the scenario where `run_relax` is `False`?